packages/blocks/src/paragraph-block/paragraph-block.ts

```diff
@@ -103,11 +107,15 @@ export class ParagraphBlockComponent extends BlockElement<
     )
       return;
 
+    const selection = this.host.selection.find('text');
+    const isCollapsed =
+      selection instanceof TextSelection ? selection.isCollapsed : false;
```

Comment:
If you use `find` method, you won't need to check the type of it, so it can be replaced by `selection?.isCollapsed ?? false`.

Reply:
Got it! I will update this change in another PR, thanks for your advice.